package.json

```diff
@@ -29,7 +29,9 @@
     "prebuild": "yarn clean",
     "build": "tsup",
     "build:watch": "yarn build --watch",
-    "test": "jest",
+    "test:unit": "jest",
+    "test:playwright": "playwright test",
+    "test:server": "node __playwright-tests__/server",
```

Comment:
Nice, thanks for getting e2e tests started!

Reply:
👍